transfer_test.go

```diff
@@ -39,14 +39,11 @@ func TestTransferDestination_UnmarshalJSON(t *testing.T) {
 
 	// Unmarshals from a JSON object
 	{
-		// We build the JSON object manually here because TransferDestination
-		// has a custom MarshalJSON implementation as well, and it'll turn into
-		// a string if we marshaled a struct instance. This ensures that we're
-		// working with a JSON objects.
-		data := []byte(`{"id":"acct_123"}`)
+		v := TransferDestination{ID: "acct_123"}
```

Comment:
This change makes it more similar to some of the other test cases in the same file, and manually crafting JSON by hand is usually not that good of an idea anyway.